components/app.js

```diff
@@ -12,11 +13,35 @@ class App {
     var newAverage = total / grades.length;
     this.pageHeader.updateAverage(newAverage)
   }
-  constructor(gradeTable, pageHeader) {
+  createGradeError(error) {
+    console.log(error)
+  }
+  handlecreateGradeSuccess() {
+    this.getGrades
+  }
+  createGrade(name, course, grade) {
+    $.ajax({
+      headers: {
+        "X-Access-Token": "QL4ulI2r"
+      },
+      url: 'https://sgt.lfzprototypes.com/api/grades',
+      method: 'POST',
+      data: {
+        name: name,
+        course: course,
+        grade: grade
+      },
+      error: this.createGradeError,
+      success: this.handlecreateGradeSuccess
+    })
+  }
+  constructor(gradeTable, pageHeader, gradeForm) {
     this.handleGetGradesError = this.handleGetGradesError.bind(this);
     this.handleGetGradesSuccess = this.handleGetGradesSuccess.bind(this)
+    this.createGrade = this.createGrade.bind(this)
     this.gradeTable = gradeTable
     this.pageHeader = pageHeader
+    this.gradeForm = gradeForm
   }
```

Comment:
Seems like you're missing a couple things:

---

![Screenshot_3](https://user-images.githubusercontent.com/16143771/100182026-7a7b4780-2e90-11eb-9c07-9cb7a81a34e4.jpg)